src/views/Profile/Profile.tsx

```diff
@@ -25,6 +29,24 @@ const Profile = () => {
     }
   }, [isLoading, isAuthenticated, user_profile, history]);
 
+  const handleResumeUploadReady = () => {
+    if (uploadRef.current.files.length !== 1 ||
+      (!uploadRef.current.files[0].name.endsWith(".pdf") &&
+      !uploadRef.current.files[0].name.endsWith(".doc") &&
+      !uploadRef.current.files[0].name.endsWith(".docx"))) return alert("Please make sure you upload a single file ending in .pdf, doc, or docx");
```

Comment:
This might be our one chance to ensure people only submit `.pdf` files.

Reply:
I agree. For sake of the companies who would receive the resumes, I believe forcing PDF on upload would be better.

Reply:
Companies already accept doc and docx so why not have them accept it through our system?